test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -52,12 +52,13 @@
     </PackageReference>
 
 
-<!--moq and deps - copied to test_dependencies folder-->
-        <PackageReference Include="Moq" Version="4.18.4" GeneratePathProperty="true" />
-        <PackageReference Include="Castle.Core" Version=" 5.1.1" GeneratePathProperty="true" />
-        <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" GeneratePathProperty="true" Condition=" '$(TargetFramework)' == 'net48' " />
+    <!--moq and deps - copied to test_dependencies folder-->
+    <PackageReference Include="Moq" Version="4.18.4" GeneratePathProperty="true" />
+    <PackageReference Include="Castle.Core" Version=" 5.1.1" GeneratePathProperty="true" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" GeneratePathProperty="true" Condition=" '$(TargetFramework)' == 'net48' " />
 
-        <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="JUnitTestLogger" Version="1.1.0" />
```

Comment:
New package reference for tests - JUnitTestLogger
This logger does not log std output and std err in the reports

Reply:
How is this used?

Reply:
Dotnet test command loads the logger dll from the app output path. Based on the name specified logger: junit